src/app/pages/rpc-sign-psbt/rpc-sign-psbt.tsx

```diff
@@ -49,7 +49,7 @@ function useRpcSignPsbt() {
       return getDecodedPsbt(psbtHex);
     },
     onSignPsbt() {
-      if (!isUndefined(signAtIndex)) {
+      if (signAtIndex.length) {
```

Comment:
wonder why there was such condition? could there be cases when `signAtIndex` is undefined?

Reply:
No, because it's an array and we pull via `params.getAll()` it always returns `[]` even if user left undefined